client/src/components/related_products/ProductCard/styles.jsx

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+
+export const CardWrapper = styled.div`
+  border: 0.75px solid;
+  display: flex;
+  flex-direction: column;
+  width: 220px;
+  height: 350px;
+  border-radius: 5px;
+  box-shadow: 0 0 20px rgba(0, 0, 0, .05), 0 0px 40px rgba(0, 0, 0, .08);
+  box-sizing: border-box;
+`;
+
+export const ProductContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 35%;
+`;
+
+export const ProductName = styled.h4`
+  box-sizing: border-box;
+  height: 0px;
```

Comment:
Setting height at 0px might make these elements not visible.

Reply:
Nice catch! I was testing and left that in there. Thank you!